MediaBrowser.Common/Extensions/BaseExtensions.cs

```diff
@@ -30,10 +30,13 @@ public static string StripHtml(this string htmlString)
         /// <returns><see cref="Guid" />.</returns>
         public static Guid GetMD5(this string str)
         {
+#pragma warning disable CA5351
```

Comment:
what is this warning?

Reply:
Do Not Use Broken Cryptographic Algorithms
https://docs.microsoft.com/en-us/visualstudio/code-quality/ca5351?view=vs-2019